treeherder/webapp/api/auth.py

```diff
@@ -16,3 +28,39 @@ def hawk_lookup(id):
         'key': str(credentials.secret),
         'algorithm': 'sha256'
     }
+
+
+class TaskclusterAuthViewSet(viewsets.ViewSet):
```

Comment:
These API endpoints probably need some tests? (We'll need to mock out parts to prevent the requests to taskcluster login)